### ISSUE TITLE
Hide Mappings from Differential Table

### ISSUE THREAD

**Issue by atruskot** [HL7/fhir-ig-publisher#616]
Based on [this discussion](https://chat.fhir.org/#narrow/stream/179252-IG-creation/topic/Hiding.20Mappings.20from.20Differential.20Table/near/328531209). 

For some Implementation Guides, it doesn't make sense to have profiled elements appear on the diff table if the element is only different from the derived structure as a result of a mapping. For example, the following object in the `StructureDefinition.differential.element.mapping` should not result in adding a new row to the diff table. 

```
{
        "id": "Appointment.cancelationReason.coding.code",
        "path": "Appointment.cancelationReason.coding.code",
        "mapping": [
          {
            "identity": "123",
            "map": "abc"
          }
        ]
},
``` 

A new setting for the IG Publisher should be created to avoid confusion on profiles where mappings shouldn't result in having an element show on the diff table.

**Comment by lmckenzi** [comment]
We think the issue isn't the element showing up in the table, but rather the current table view doesn't tell you what's happened to the element.  We propose to add icons or small labels on the differential table if there are changes that have happened that only manifest on the data dictionary page or mappings tab with links to the appropriate section of the appropriate tab.  This will ensure that if something shows up on the differential summary table, it's always obvious as to 'why'.  (And it may prompt the reader to go look at the other tabs to see what's changed.)

Would making this change address the concern @atruskot?

**Comment by atruskot** [comment]
@lmckenzi, yes, I definitely that would be a helpful addition and would clarify why an element is showing up. 

As a note, the diff table doesn't currently populate many of the existing flags (such as summary, modifier, etc.). If I recall correctly, it currently only displays the must-support and constraint flags. This may be a good time to consider what flags should be displayed on this table.

**Comment by lmckenzi** [comment]
summary and modifier can't ever be changed, so there wasn't much point in including them in the differential table.   Both indicators are locked down in the resource design.

**Comment by lmckenzi** [comment]
We will add the proposed icons to the summary table to indicate whether there have been changes to data dictionary and/or mapping content that isn't visible from the summary table.  The links will have flyovers explaining what they mean and will hyperlink to the appropriate element in the mapping or data dictionary tab.
